src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,72 @@
+import cn from 'classnames';
+import { Statuses } from '../../types/Statuses';
+
+interface Props {
+  status: Statuses;
+  setStatus: React.Dispatch<React.SetStateAction<Statuses>>;
+  isAnyTodosCompleted: boolean;
+  notCompletedTodosCount: number;
+  onDeleteCompleted: () => void;
+}
+
+export const Footer: React.FC<Props> = ({
+  status,
+  setStatus,
+  isAnyTodosCompleted,
+  notCompletedTodosCount,
+  onDeleteCompleted,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${notCompletedTodosCount} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={cn('filter__link', {
+            selected: status === 'all',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={() => setStatus('all')}
```

Comment:
all statuses need to move in constants or Enum ( best way Enum), then use in all places in your application.